packages/stark-ui/assets/themes/_app-header-theme.scss

```diff
@@ -1,165 +1,143 @@
-  line-height: 48px;
-  margin-left: 56px; /* icon button width 48px plus 8px between elements is 56px; */
-}
-
-@media #{$tablet-screen-query} {
-  .stark-header-mobile,
-  .stark-header-expanded {
-    display: none;
-  }
-
-  /* Compact Header */
-  .stark-header-compact {
+  .stark-app-bar {
     display: flex;
-    margin-left: 56px;
-    margin-right: 56px;
-  }
+    justify-content: space-between;
+    padding-top: 6px;
 
-  /* Expanded Header */
-  .full-curtain {
-    & .stark-header-compact {
+    .stark-content-left {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      width: 50%;
+      > div {
+        display: flex;
+        justify-content: flex-start;
+        &.stark-actions {
+          margin-left: -10px;
+          .stark-home-button {
+            display: none;
+          }
+        }
+        &.stark-page-title {
+          margin-top: 5px;
+          h1 {
+            font-size: 32px;
+            font-weight: 200;
+          }
+        }
+        > * {
+          margin-right: 3px;
+        }
+      }
+    }
+    .stark-content-center {
       display: none;
+      width: 0%;
     }
-    & .stark-header-expanded {
+    .stark-content-right {
       display: flex;
-      margin-right: 0;
-      padding-right: 56px;
+      flex-direction: column;
+      width: 50%;
+      > div {
+        display: flex;
+        justify-content: flex-end;
+        margin-bottom: 5px;
+        &.stark-actions-alt {
+          position: fixed;
+          right: 15px;
+          bottom: 15px;
+        }
+        &:last-child {
+          margin-bottom: 0;
+        }
+        > * {
+          margin-left: 3px;
+        }
+      }
     }
   }
 }
 
-/* IE flex layout fix */
-.stark-header-left {
-  margin-right: auto;
-}
-
-.stark-header-right {
-  margin-left: auto;
-}
-.main-header_Right {
-  line-height: 14px;
-}
-.expanded-header-right {
-  display: flex;
-  justify-content: flex-end;
-}
-.expanded-header-right-zone1 {
-  display: inline-flex;
-}
-.expanded-header-right-zone2 {
-  display: inline-flex;
-  flex-direction: column;
-  margin-left: 15px;
-}
-.app-title {
-  font-size: 20px;
-  line-height: 48px;
-}
-
-.stark-page-header {
-  position: relative;
-}
-.stark-page-title {
-  font-size: 32px;
-  font-weight: 200;
-  line-height: 1.1;
-  padding-bottom: 15px;
-  padding-left: 8px;
-}
-.stark-page-header-actions {
-  position: fixed;
-  bottom: 15px;
-  top: auto;
-  right: 15px;
-  & .md-button {
-    margin: 0;
-  }
-}
-.stark-header {
-  overflow: visible;
-  background: mat-color(map-get($base-theme, primary-palette), 900) url(/assets/images/app-header.png) top right no-repeat;
-  color: #fff;
-  height: $stark-header-size;
-  & .stark-container {
-    position: relative;
-    height: 100%;
-  }
-  & a {
-    color: #fff;
+@media #{$tablet-query} {
+  .stark-app-header {
+    height: $stark-header-size-desktop;
+    .stark-app-bar {
+      .stark-content-left {
+        width: 33%;
```

Comment:
33% width will not be enough for long page titles...

![header - long page title issue](https://user-images.githubusercontent.com/5698318/43636756-4534464c-9713-11e8-84db-4c60a07882ab.PNG)